emmet-core/emmet/core/openff/core.py

```diff
@@ -0,0 +1,9 @@
+from emmet.core.openff import MoleculeSpec
+from emmet.core.base import EmmetBaseModel
+
+
+class ClassicalMDDoc(EmmetBaseModel):
```

Comment:
there is a class definition for SolvationDoc that is commented out in "emmet-core/emmet/core/openff/solvation.py" (line 30), outside of that I don't see other references to this ClassicalMDDoc model.

Should this be used? If not, should it be removed?

Reply:
This is a bit aspirational so I can remove it for now. The intent is for this class to eventually contain the results of all the different builders.

Reply:
okay, good to know. can be re-implemented in the future when you have that fleshed out